src/components/EventForm.js

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import "react-dates/initialize";
+import "react-dates/lib/css/_datepicker.css";
+import { DateRangePicker } from "react-dates";
+import moment from "moment";
+import uuid from "uuid";
+
+export default class EventForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      title: "",
+      description: "",
+      organisator: "",
+      city: "",
+      category: "",
+      image: false,
+      id: uuid(),
+      createdAt: moment(),
+      startDate: moment(),
+      startDateId: "",
+      endDate: moment(),
+      endDateId: "",
+      calendarFocused: null,
+    };
+  }
+
+  onTitleChange = e => {
+    const title = e.target.value;
+    this.setState(() => ({ title }));
+  };
+
+  onDescriptionChange = e => {
+    const description = e.target.value;
+    this.setState(() => ({ description }));
+  };
+
+  onOrganisatorChange = e => {
+    const organisator = e.target.value;
+    this.setState(() => ({ organisator }));
+  };
+
+  onCityChange = e => {
+    const city = e.target.value;
+    this.setState(() => ({ city }));
+  };
+
+  onCategoryChange = e => {
+    const category = e.target.value;
+    this.setState(() => ({ category }));
+  };
+
+  onImageChange = e => {
```

Comment:
Hm, you have exactly the same method declared 6 times, so it would be good to change it to something generic. What about something like this:
```
onChange = (e, key) => {
  const { value } = e.target
  this.setState({ [key]: value })
}
```
? As `key` prop you will pass type of data which you want to change i.e. `category` or `city`. It would be less code, more generic and DRY in general :)